Makefile

```diff
@@ -75,11 +102,24 @@ clean: # Clean up project
 	make stop
 	docker network rm $(DOCKER_NETWORK) 2> /dev/null ||:
 
+run-smoke-test:
+	eval "$$(make aws-assume-role-export-variables)"
+	make run-smoke COGNITO_POSTCODE_AUTH=$$(make -s project-aws-get-admin-secret | jq .POSTCODE_PASSWORD | tr -d '"')
```

Comment:
Could we set the COGNITO_POSTCODE_AUTH parameter inside the run-smoke make target rather than have it passed through?